src/hooks/incomplete.ts

```diff
@@ -30,7 +32,17 @@ const hook: Hook.CommandIncomplete = async function ({ config, matches, argv })
     return config.runCommand('help', [toStandardizedId(command, config)]);
   }
 
-  return config.runCommand(toStandardizedId(command, config), argv);
+  const standardCommand = toStandardizedId(command, config);
```

Comment:
@cristiand391 I believe toStandardizedId is correct. @oclif/core maintains ids in standard format and if the id would contain spaces when passed to [runCommand](https://github.com/oclif/core/blob/main/src/config/config.ts#L326) the [findCommand](https://github.com/oclif/core/blob/main/src/config/config.ts#L374) function would fail, as it assumes the provided id is in standard format.

Reply:
@cristiand391 But I should display the configured command in the message.